src/components/Table/Table.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { variant } from 'styled-system';
+import theme from '../../theme';
+import { get } from '../../utils/utils';
+import { COLOR, COMMON, TYPOGRAPHY } from '../../utils/constants';
+
+import TableBody from './TableBody';
+import TableCell from './TableCell';
+import TableHead, { TableHeadWrapper } from './TableHead';
+import TableRow, { TableRowWrapper } from './TableRow';
+
+
+const Table = styled.table`
+    background-color: ${get('colors.white')};
+    text-align: center;
+    border-collapse: collapse;
+    width: 100%;
+    
+    ${TableHeadWrapper} ${TableRowWrapper} {
+        ${variant({
+    prop: 'type',
+    scale: 'table',
+  })}
```

Comment:
I know I mentioned to use variant, but it does seem a bit weird because we have three calls to variant in this file for just three styles from the theme.js. So perhaps that means this isn't necessarily a situation where variant is well-suited.

Did some playing around and confirmed you can replace this call to variant with

```Javascript
border-bottom: ${(props) => props.type === 'primary' && `${get('table.primary.borderBottom')(props)}`};
```

Which would make it more clear that this style is applied if it is a `primary` table and there are no other variants just a boolean of this table is primary or not primary

Reply:
now this, is epic